lib/aws/ses.rb

```diff
@@ -0,0 +1,27 @@
+##
+# ActionMailer delivery method for SES inspired by https://github.com/drewblas/aws-ses
+#
+module Aws
+  module SES
+    class Base
+      def initialize(*args); end
+
+      def deliver!(mail)
+        response = ses_client.send_raw_email(
+          raw_message: {
+            data: mail.to_s,
+          },
+          destinations: [mail.to, mail.cc, mail.bcc].flatten.compact
```

Comment:
based on my reading of the API, `destinations` is not required? since the to, cc, etc are all in the raw mail body? http://docs.aws.amazon.com/ses/latest/APIReference/API_SendRawEmail.html

Reply:
You're right. For some reason I thought it was. Will fix